Lib/test/clinic.test.c

```diff
@@ -5358,19 +5361,15 @@ Test___init__(PyObject *self, PyObject *args, PyObject *kwargs)
         !_PyArg_NoKeywords("Test", kwargs)) {
         goto exit;
     }
-    _return_value = Test___init___impl((TestObj *)self);
-    if ((_return_value == -1) && PyErr_Occurred()) {
-        goto exit;
-    }
-    return_value = PyLong_FromLong(_return_value);
```

Comment:
Note: this particular test is wrong anyway. I've suggested to disallow overriding return converters for `__init__` functions in #116171.

Reply:
Merged.